api/clients/v2/payloaddispersal/payload_disperser.go

```diff
@@ -169,6 +170,9 @@ func (pd *PayloadDisperser) SendPayload(
 
 	err = pd.certVerifier.CheckDACert(timeoutCtx, eigenDACert)
 	if err != nil {
+		// TODO(samlaf): do we want to return an api.ErrorFailover here? In case the disperser is being malicious
+		// and returning invalid data? This could also just indicate that our cert building procedure above is buggy though...
+		// but perhaps also a reason to failover?
 		return nil, fmt.Errorf("verify cert for blobKey %v: %w", blobKey.Hex(), err)
```

Comment:
thoughts here?

Reply:
I think failover probably makes sense.

We could do some more fine-grained logic like you mentioned elsewhere, where we only failover if this happens multiple times... but it doesn't seem worth it right now

Reply:
Wow this was perfectly planted to be solved by https://github.com/Layr-Labs/eigenda/pull/1938.
Once that merges we'll map the `BUG` status code to failover here.

Reply:
[7326b9a](https://github.com/Layr-Labs/eigenda/pull/1924/commits/7326b9a55d1fd4e25faa566451983ea729efd6e5)
Actually opted to failover on both INVALID_CERT and INTERNAL_ERROR status codes..